packages/@vue/cli-plugin-eslint/lint.js

```diff
@@ -138,9 +177,19 @@ function normalizeConfig (args) {
   const config = {}
   for (const key in args) {
     if (renamedArrayArgs[key]) {
-      config[renamedArrayArgs[key]] = args[key].split(',')
+      const keyPaths = [...renamedArrayArgs[key]]
+      const lastKey = keyPaths.pop()
+      for (const k of keyPaths) {
+        config[k] = {}
+      }
+      config[lastKey] = args[key].split(',')
```

Comment:
env: ['overrideConfig', 'env'],
global: ['overrideConfig', 'globals'],

`--env browser --global window` gets normalized to `{ overrideConfig: {}, env: ['browser'], globals: ['window'] }`. Is this the intended result?
Because in my understanding, `env` and `globals` should be put under `overrideConfig`.

Reply:
That's a bug. sorry I will fix it :sweat:

Reply:
Thank you for your review!
I changed this PR. Could you check it again?